modules/service/src/main/scala/lucuma/itc/input/customSed/CustomSedOdbAttachmentResolver.scala

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.input.customSed
+
+import cats.effect.Resource
+import cats.effect.kernel.Async
+import cats.syntax.all.*
+import fs2.io.net.Network
+import fs2.text
+import org.http4s.AuthScheme
+import org.http4s.Credentials
+import org.http4s.Headers
+import org.http4s.Request
+import org.http4s.Uri
+import org.http4s.client.Client
+import org.http4s.ember.client.EmberClientBuilder
+import org.http4s.headers.Authorization
+import org.typelevel.log4cats.Logger
+
+/**
+ * Implementation of `CustomSed.Resolver` that uses a URL to fetch the custom SED.
+ */
+class CustomSedOdbAttachmentResolver[F[_]: Async: Logger] private (
+  client:     Client[F],
+  odbBaseUrl: Uri,
+  authToken:  String
+) extends CustomSedDatResolver[F]:
+
+  protected def datLines(id: CustomSed.Id): F[fs2.Stream[F, String]] =
+    val uri: Uri            =
+      odbBaseUrl / "attachment" / id.programId.show / id.attachmentId.show
+    val request: Request[F] =
+      Request(
+        uri = uri,
+        headers = Headers:
+          Authorization:
+            Credentials.Token(AuthScheme.Bearer, authToken)
+      )
+
+    for _ <- Logger[F].info(s"Fetching custom SED for id [$id]: $uri")
```

Comment:
`Logger[F].....map{_ => client...}`?

Reply:
Yes, or `>>`, I had more things in the `for` but then deleted them. I'll simplify.